reactioncommerce/reaction-admin
# Update to Meteor 2.0, React 16.14.0 and enable HMR

Impact: **minor**
Type: **performance|chore**

## Issue
Meteor was outdated as version 2.0 recently came out. This version introduces HMR with React Fast Refresh, making for a better developer experience when working on the front-end.

## Solution
Update to Meteor 2.0. Also update to React 16.14.0 as HMR needs React > 16.9.0 to work and we were running 16.8.6.

## Breaking changes
None.


## Testing
1. Start `reaction-admin`, check that the UI is working as it should.
2. Make a change to a React component.
3. Check that the change takes effect through HMR rather than a page reload.
